code/modules/reagents/reagents/medicine.dm

```diff
@@ -864,9 +864,12 @@ datum/reagent/medicine/respirodaxon/affect_blood(var/mob/living/carbon/M, var/al
 /datum/reagent/medicine/haloperidol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
 	if(M.bloodstr)
 		for(var/current in M.bloodstr.reagent_list)
-			var/datum/reagent/toxin/pararein/R = current
+			var/datum/reagent/toxin/R = current
 			if(!istype(R, src))
-				R.remove_self(effect_multiplier * pick(list(1,2,3)))
+				R.remove_self(effect_multiplier * pick(list(2,4,6)))
+			var/datum/reagent/stim/S = current
+			if(!istype(S, src))
+				S.remove_self(effect_multiplier * pick(list(2,4,6)))
 	var/effective_dose = dose
 	if(issmall(M))
 		effective_dose *= 2
```

Comment:
line 882 shows "else if(effective_dose < 3)"
perhaps make it so its above 11?
Atm is a really really affective as a sleeping chem and will likely be used in 5-10u doses. Meaning its going to sleep a lot of people and be annoying to use even in controlled areas like medical

Reply:
I like it at 3